### PR TITLE
Tests with mock server, parsing from .pcap file

### PR DESCRIPTION
**Pull request by yan-sh**
- Add tests for mocking server
- Add tests for parsing responses from .pcap file (path to the file must be specified in the PCAP_TEST_FILE_PATH environment variable)

### CODE REVIEW

**Comment by nikita-volkov** [comment]
You've accidentally pushed files from dist-newstyle directory.

You can add the following gitignore to your project root to avoid that in the future.
```
.gitignore
dist-newstyle/
```

Please don't push the .gitignore file itself though.

**Comment by yan-sh** [comment]
> You've accidentally pushed files from dist-newstyle directory.
> 
> You can add the following gitignore to your project root to avoid that in the future.
> 
> ```
> .gitignore
> dist-newstyle/
> ```
> 
> Please don't push the .gitignore file itself though.

Oh, my bad, i've just update

File: library/OpcXmlDaClient.hs [hunk 1]
```diff
@@ -87,7 +91,8 @@ encDecOp encode decode manager (RequestTimeout timeout) (Uri request) input =
           | Just exc <- fromException @IOException exc ->
             return $ Left $ IoError exc
           | otherwise -> throwIO exc
-        Right response ->
+        Right response -> do
+          -- BS.putStrLn $ BL.toStrict (Hc.responseBody response)
```
Comment (nikita-volkov) [inline library/OpcXmlDaClient.hs:95]: Commented code

**Review by nikita-volkov — CHANGES_REQUESTED** [review]

File: library/OpcXmlDaClient.hs (hunk 1 above)
Comment (yan-sh) [reply library/OpcXmlDaClient.hs]: Remove all code with debug-print

**Pull request merged**